AdobeStockImageAdminUi/view/adminhtml/web/js/signIn.js

```diff
@@ -51,11 +58,36 @@ define([
                         resolve(response);
                     }.bind(this))
                     .catch(function (error) {
-                        reject(error);
-                    });
+                    this.showPopupOnFail ?
+                         this.getLoginErrorPopup()
+                         : reject(error);
+                }.bind(this));
             }.bind(this));
         },
 
+        /**
+         * Show popup that user failed to login.
+         */
+        getLoginErrorPopup: function () {
+            confirm({
+                title: $.mage.__('The user is not able to login'),
+                // eslint-disable-next-line max-len
+                content: $.mage.__('Login failed. Check if the Secret Key  entered <a href="' + this.settingsUrl + '">Configuration → System → Adobe Stock Integration</a> correctly and try again'),
```

Comment:
Double space in message. Actually it would be better to pass the message from the backend (as it can vary depending on situation)
```suggestion
                content: $.mage.__('Login failed. Check if the Secret Key entered <a href="' + this.settingsUrl + '">Configuration → System → Adobe Stock Integration</a> correctly and try again'),
```